deploy/patch.yaml

```diff
@@ -29,4 +30,4 @@ spec:
         name: tmp
       - name: cfm
         configMap:
-          name: kepler-estimator-cfm
\ No newline at end of file
+          name: kepler-estimator-cfm
```

Comment:
This will be changed to `kepler-cfm` to have a single point of configuration.
The rest is /lgtm.